tests/cases/conformance/jsdoc/jsdocStringLiteral.ts

```diff
@@ -0,0 +1,12 @@
+// @allowJs: true
+// @filename: in.js
+// @out: out.js
+/**
+ * @param {'literal'} p1
+ * @param {"literal"} p2
+ * @param {'literal' | 'other'} p3
+ * @param {'literal' | number} p4
+ */
+function f(p1, p2, p3, p4) {
+    return p1 + p2 + p3 + p4 + '.';
```

Comment:
Done. 
Hilariously, `@Filename` is case-sensitive in fourslash tests, but not compiler and conformance tests.
Also, Codeflow has **two** Give Feedback buttons, and they are not the same -- one is the inverse colour of the other.
OK, I'm done complaining now.